pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -746,7 +746,7 @@ func (c *VolumeZoneChecker) predicate(pod *v1.Pod, meta PredicateMetadata, nodeI
 func GetResourceRequest(pod *v1.Pod) *schedulernodeinfo.Resource {
 	result := &schedulernodeinfo.Resource{}
 	for _, workload := range pod.Spec.Workloads() {
-		result.Add(workload.Resources.Requests)
+		result.Add(workload.ResourcesAllocated)
```

Comment:
this change should be guarded by the featureGate()

Reply:
fixed.